### PR TITLE
Use `all_linters()` instead of using tags

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: .lintr [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-    tags = NULL,
+linters: all_linters(
     commented_code_linter = NULL,
     cyclocomp_linter = cyclocomp_linter(40),
     fixed_regex_linter = NULL,
     function_argument_linter = NULL,
     indentation_linter = NULL,
     line_length_linter(120L),
     namespace_linter = NULL,
-    nested_ifelse_linter = NULL,
```
Comment (IndrajeetPatil) [inline .lintr:10]: We should be linting this.

File: .lintr [hunk 2]
```diff
@@ -1,13 +1,11 @@
-linters: linters_with_tags(
-    tags = NULL,
+linters: all_linters(
```
Comment (IndrajeetPatil) [inline .lintr:1]: Just a more intuitive API to access all offered linters.

File: .lintr (hunk 1 above)
Comment (lorenzwalthert) [reply .lintr:10]: Sure. Wanna add it and fix the lints in this PR?

Comment (IndrajeetPatil) [reply .lintr:10]: The fact that it's not `NULL` means it's already included. There are no new lints to fix. :)

Comment (lorenzwalthert) [reply .lintr:10]: Great 😊. But then I don’t get why you said we should be listing this in the first place 🤔

Comment (IndrajeetPatil) [reply .lintr:10]: When a linter is set to NULL, it is excluded from _all_ linters, and thus not used. So I am just removing its exclusion.

**Review by lorenzwalthert — APPROVED** [review]
Thanks.

**Pull request merged**